docs/csharp/tutorials/working-with-linq.md

```diff
@@ -257,8 +257,9 @@ Briefly, lazy evaluation states that the evaluation of a statement is not perfor
 
 Remember that we generated the original deck using a LINQ query. Each shuffle is generated by performing three LINQ queries on the previous deck. All these are performed lazily. That also means they are performed again each time the sequence is requested. By the time you get to the 52nd iteration, you're regenerating the original deck many, many times. Let's write a log to demonstrate this behavior. Then, you'll fix it.
 
-Here's a log method that can be appended to any query to mark that the query executed.
+In your `Extensions.cs` file, type in or copy the method below. This new extension method will create a new file called `debug.log` within your project's directory and record what query is currently being executed to the log file. This extension method can be appended to any query to mark that the query executed.
 
+*`Extensions.cs`*
```

Comment:
This can be removed. It doesn't conform to our standard presentation, which I think makes it confusing. The text also explains that this code goes in Extensions.cs.

Reply:
Sure. I think I added some more of these in my previous PR for this repo, so I'll make sure it's uniform.